src/test/java/no/uib/inf112/core/player/FlagRegistrationTest.java

```diff
@@ -1,20 +1,100 @@
 package no.uib.inf112.core.player;
 
+import no.uib.inf112.core.GameGraphics;
+import no.uib.inf112.core.RoboRally;
+import no.uib.inf112.core.map.cards.Movement;
+import no.uib.inf112.desktop.TestGraphics;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
+import java.io.File;
+
 import static org.junit.Assert.*;
 
-public class FlagRegistrationTest {
+public class FlagRegistrationTest extends TestGraphics {
 
-    Player player = new NonPlayer(0, 0, Direction.NORTH);
+    private Player player = new NonPlayer(0, 0, Direction.NORTH);
+    private static RoboRally roborally;
 
-    //TODO #77 this should be tested way more.
+    @BeforeClass
+    public static void beforeClass() {
+        roborally = GameGraphics.getRoboRally();
+        roborally.changeMap(TEST_MAP_FOLDER + File.separatorChar + "flag_test_map.tmx");
```

Comment:
What about changing the GameGraphics to include RoboRally newRoboRally(String map), i think this kind of creation is not intuitive